pkgs/development/haskell-modules/with-packages-wrapper.nix

```diff
@@ -46,14 +47,18 @@ let
   llvm          = lib.makeBinPath
                   ([ llvmPackages.llvm ]
                    ++ lib.optional stdenv.isDarwin llvmPackages.clang);
+  ghcLibdirLink = runCommand "ghc_libdir" {} ''
+    mkdir -p '${libDir}'
+    echo '${ghcLibdir}' > '${libDir}/ghc_libdir_override'
```

Comment:
I assume this somehow related to `ghc_libdir` above; what does `ghc_libdir_override` do?